config_files/ps-extractor.sh

```diff
@@ -1,14 +1,22 @@
 #!/bin/bash
 
 folder=$1
-destination="/var/www/html"
 
-if [[ -n "$folder" ]]; then
-    if [ -d $folder/prestashop ]; then
-        cp -n -R $folder/prestashop/* $destination
-    else
-        unzip -n -q $folder/prestashop.zip -d $destination
+if [[ -n "$folder" ]]; then  
+
+    # dwl version contains zip file with tree structure (1.7)
+    if [ ! -d $folder/prestashop ]; then
+        unzip -n -q $folder/prestashop.zip -d $folder/prestashop
+	rm -rf $folder/prestashop.zip	
     fi
+
+    # prepair tree structure for volumes
+    mv $folder/prestashop/themes $folder/prestashop/modules $folder/prestashop/override $folder/
+
+    # build relative symlinks for volumes
+    ln -s ../themes $folder/prestashop/themes
+    ln -s ../modules $folder/prestashop/modules
+    ln -s ../override $folder/prestashop/override
```

Comment:
It will be difficult because we can't mount volume inside volume in Docker.
I will try to manage the dir inside the /var/www/html but the admin folder and the install folder are "variables" on execution and we can only declare volumes on build. I think we have no choice to manage the full /var/www/html with no volumes outside :(